File: C_Sharp_code/Example/WindowsFormsApp/VB_demo19/Animal.vb
Public Class Animal
    Public Sub Eat()
        Console.WriteLine("Animal Eat")
    End Sub
    Public Function Speak() As String
        Return "Animal Speak"
    End Function
End Class

Class Lion
    Inherits Animal
End Class

Class Horse : Inherits Animal
    'VB中:表示換行
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo18/Animal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CS_demo18
{
    class Animal
    {
        public void eat()
        {
            Console.WriteLine("Animal Eat");
        }
        public string speak()
        {
            return "Animal Speak";
        }
    }
    class Lion : Animal
    {

    }
    class Horse : Animal { }
}
